protostar/commands/cairo1_commands/build_cairo1_command.py

```diff
@@ -67,7 +67,7 @@ async def _build_contract(
             f"Multiple files found for contract {contract_name}, "
             f"only one file per contract is supported in cairo1!"
         )
-        cairo1.compile_starknet_contract_to_casm_from_path(
+        cairo1.compile_starknet_contract_to_sierra_from_path(
```

Comment:
I don't understand this change, is it accidental?

Reply:
Leftover from local testing.